pkg/service/auth.go

```diff
@@ -32,7 +32,8 @@ func NewAuthService(db *gorm.DB, jwtTokenService JWTTokenService) AuthService {
 //
 // External users are not allowed to use password authentication.
 // TODO(kolesnikovae): It's true for "some" authentication providers.
-//  Others may need us to pass through the credentials (e.g. LDAP).
+//
+//	Others may need us to pass through the credentials (e.g. LDAP).
```

Comment:
"Others may need us to pass through the credentials (e.g. LDAP)." is part of the TODO note

Reply:
there was a tab that made the formatter go nuts, fixed